components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -537,6 +548,16 @@ fn insert_local_login(conn: &Connection, local_login: &LocalLogin) -> Result<()>
         )";
 
     let login = &local_login.login;
+    let dec_login = &local_login.login.clone().decrypt(encdec)?;
+
+    match new_db.check_for_dupes(&login.guid(), &dec_login.entry(), &encdec) {
+        Ok(_) => {}
+        Err(e) => {
+            log::warn!("Duplicate {} ({}).", login.record.id, e);
+            // Should we record this as an error? or just silently ignore it
```

Comment:
I think it might as well be recorded as an error - probably just have this return a bool?